RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# How to release
+
+Perform the following steps in order to release new verions of helm chart.
+
+1. Prepare and merge PR with the following changes:
+
+   - update [changelog][changelog]
+   - update [chart][chart]
+   - update [deploy/README.md][deploy] (support matrix)
+
+1. Branch out `release-v${TAG}` and add the following changes:
+
+   - update [deploy/README.md][deploy] (title)
```

Comment:
Which title do you mean?

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/README.md#deployment-guide-for-unreleased-version

Should I link it specifically?

Reply:
It wasn't clear to me, so maybe we should.